Tests/NetworkServiceTests/NetworkErrorTests.swift

```diff
@@ -0,0 +1,19 @@
+import NetworkService
+import XCTest
+
+final class NetworkErrorTests: XCTestCase {
```

Comment:
I'd recommend to test if the enum conforms to `Error`.

Reply:
Thanks for the suggestion. Test was added.